### PR TITLE
Corrected 'Meeting Room' license name

### PR DESCRIPTION
**Pull request by shivtorov**
Issue #3413

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/room-systems/skype-room-systems-v2.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Microsoft has a dedicated SKU for licensing meetings and calling on a per-device
 > If you are currently using E1, E3, E4, E5 SKUs with Skype for Business Standalone Plan 1 or Skype for Business Plan 2 or Skype for Business Plan 2 with Audio conferencing or with Office365 Phone System and a Calling Plan, these will continue to work. However, you should consider moving to a simpler licensing model in the table above after current licenses expire.
```
Comment (JohanFreelancer9) [inline Teams/room-systems/skype-room-systems-v2.md]:
```suggestion
> If you are currently using E1, E3, E4, or E5 SKUs with Skype for Business Standalone Plan 1 or Skype for Business Plan 2 or Skype for Business Plan 2 with Audio conferencing or with Office 365 Phone System and a Calling Plan, these will continue to work. However, you should consider moving to a simpler licensing model in the table above after current licenses expire. 
```

Comment (dstrome) [reply Teams/room-systems/skype-room-systems-v2.md]: @shivtorov - please accept Johan's changes so we can merge this PR. thanks!

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**